presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -116,6 +116,47 @@ Memory Management Properties
     This is the amount of memory set aside as headroom/buffer in the JVM heap
     for allocations that are not tracked by Presto.
 
+
+Query Management Properties
+---------------------------
+
+``query.max-execution-time``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``duration``
+    * **Default value:** ``100 days``
+
+    The maxiumum allowed time for a query to be actively executing across the
+    cluster on all nodes, before it is terminated.
+
+``query.max-run-time``
+^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``duration``
+    * **Default value:** ``100 days``
+
+    The maxiumum allowed time for a query to be running on the cluster, before
+    it is terminated.
```

Comment:
- can be overridden with `query_max_run_time` session
- "max run time" is misnomer... this includes analysis, queuing & planning time as well. Of these, queuing is the most important and should be mentioned here. Or you can say this is "max time since query creation".